test/lib/game/game_supervisor_test.exs

```diff
@@ -0,0 +1,14 @@
+defmodule Reactor.GameSupervisorTest do
+  use ExUnit.Case, async: true
+
+  test "can create a game" do
+    %{workers: workers} = Supervisor.count_children(Reactor.GameSupervisor)
```

Comment:
Not sure you even need this test as you're testing the behavior of the gen server